app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -1,37 +1,43 @@
 import * as yup from "yup";
 
-export const NOTES_FORM_INITIAL_FORM_VALUES = {
+export const NOTES_FORM_INITIAL_VALUES = {
   title: "",
   description: "",
-  tagsList: [],
+  assignedContactId: "",
+  tag: "",
 };
 
+export const NOTES_SEED_VALUES = {
+  title: "How to Claim Warranty?",
+  description:
+    "Description Goes Here. Description Goes Here. Description Goes Here. Description Goes Here. Description Goes Here. Description Goes Here. Description Goes Here. Description Goes Here. Description Goes Here. ",
+  assignedContactId: "1",
+  tag: "Getting Started",
+  created_at: "2023-08-29T19:01:08+05:30",
```

Comment:
Please don't follow mixed naming conventions. Whatever you defined in JS side ought to be camelCased.

Reply:
Okay, will make this change.